DigitalExcellence/dex-frontend
# Feature/285 lighthouse cd

## Description

Created a new action for the lighthouse audit. 

This pull request should be blocked until #332 (improving lighthouse score) is ready. Once that pull request is ready this one can be accepted. If done otherwise, everyone creating a pull request will have one failed action in the overview. 

## Type of change


## Checklist


## Steps to Test or Reproduce

Outline the steps to test or reproduce the PR here.
These steps will be used during release testing.

1. Create a pull request to develop
2. Check the lighthouse audit in the actions tab
3. Check if the lighthouse audit is working as expected (warning for potential slow elements etc.)

## Link to issue

Closes: #285
